src/main/resources/templates/placeOrder.html

```diff
@@ -17,121 +17,121 @@
             <h1 class="display-4">Place an Order</h1>
         </div>
     </div>
-</main>
 
-<div class="container">
-    <form>
-        <div class="form-row">
-            <div class="form-group col-md-6">
-                <label for="inlineFormInputName">Name</label>
-                <input type="email" class="form-control" id="inlineFormInputName" placeholder="Company or name">
+    <div class="container outerContentWrapper">
+        <form action="#" th:action="@{/saveOrder}" th:object="${saveOrder}" method="post">
+            <div class="form-row">
+                <div class="form-group col-md-6">
+                    <label for="inlineFormInputName">Name</label>
+                    <input type="text" class="form-control" id="inlineFormInputName" placeholder="Company or name" th:field="*{customerName}">
+                </div>
             </div>
-        </div>
-        <div class="form-group">
-            <label for="inputAddress">Address</label>
-            <input type="text" class="form-control" id="inputAddress" placeholder="1234 Main St">
-        </div>
-        <div class="form-group">
-            <label for="inputAddress2">Address 2</label>
-            <input type="text" class="form-control" id="inputAddress2" placeholder="Apartment, studio, or floor">
-        </div>
-        <div class="form-row">
-            <div class="form-group col-md-6">
-                <label for="inputCity">City</label>
-                <input type="text" class="form-control" id="inputCity">
+            <div class="form-group">
+                <label for="inputAddress">Address</label>
+                <input type="text" class="form-control" id="inputAddress" placeholder="1234 Main St" th:field="*{customerAddress1}">
```

Comment:
Added the field attribute to the input element. This will allow the element values to bind to the field of the object.